app/javascript/onboarding/components/ClosingSlide.jsx

```diff
@@ -1,22 +1,35 @@
 import { h, Component } from 'preact';
 import PropTypes from 'prop-types';
 
-import { getContentOfToken } from '../utilities';
+import { updateOnboarding } from '../utilities';
 
 class ClosingSlide extends Component {
   componentDidMount() {
-    const csrfToken = getContentOfToken('csrf-token');
-    fetch('/onboarding_update', {
-      method: 'PATCH',
-      headers: {
-        'X-CSRF-Token': csrfToken,
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({ user: { last_onboarding_page: 'closing slide' } }),
-      credentials: 'same-origin',
-    });
+    updateOnboarding('closing slide');
   }
 
+  whatsNext() {
+    const properties = [
```

Comment:
please, rename this to something like `whatsNextLinks`,

or just return the array (see next comment)